src/components/input/TextComponent.tsx

```diff
@@ -61,13 +71,15 @@ const TextComponentPropTypes = {
   tooltip: PropTypes.string,
   tooltipCloseLabel: PropTypes.string,
   value: PropTypes.string,
-  showRequired: PropTypes.bool
+  showRequired: PropTypes.bool,
 };
 
 const TextComponent: React.FC<TextComponentPropsWithType> = ({
+  rightDecorators,
```

Comment:
A future improvement could be to use a "decorators" (instead of rightDecorators ) and for each decorator define with a prop which side to use (left, right).